components/OverflowMenu.js

```diff
@@ -64,8 +64,10 @@ class OverflowMenu extends Component {
   }
 
   getMenuPosition = () => {
-    const menuPosition = this.menuEl.getBoundingClientRect();
-    this.setState({ menuPosition });
+    if (this.menuEl) {
```

Comment:
@chrisdhanaraj  ?

I didn't realize it was possible for `refs` to get used before they're defined since this `getMenuPosition()` method is invoked inside `componentDidMount()` ... I don't see a difference when I remove the if statement.